weave/trace_server/migrations/006_seed_costs.up.sql

```diff
@@ -170,24 +170,24 @@ VALUES
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-pro', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.0-pro', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.0-pro-001', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.0-ultra', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.0-ultra-001', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.0-pro-002', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-pro', now(), 5e-06, 'USD', 1.5e-05, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-pro-001', now(), 5e-06, 'USD', 1.5e-05, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-pro-preview-0514', now(), 5e-06, 'USD', 1.5e-05, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-pro-preview-0215', now(), 5e-06, 'USD', 1.5e-05, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-pro-preview-0409', now(), 5e-06, 'USD', 1.5e-05, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-flash', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-flash-001', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-1.5-flash-preview-0514', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'gemini-experimental', now(), 0, 'USD', 0, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-vision-models', 'gemini-pro-vision', now(), 2.5e-07, 'USD', 5e-07, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-vision-models', 'gemini-1.0-pro-vision', now(), 2.5e-07, 'USD', 5e-07, 'USD', 'system', now()), 
-    (generateUUIDv4(), 'default', 'default', 'vertex_ai-vision-models', 'gemini-1.0-pro-vision-001', now(), 2.5e-07, 'USD', 5e-07, 'USD', 'system', now()), 
+    (generateUUIDv4(), 'default', 'default', 'vertex_ai-language-models', 'google_ai_studio-pro', now(), 5e-07, 'USD', 1.5e-06, 'USD', 'system', now()),
```

Comment:
This is very very bad. Migrations shall never be edited. Please undo these changes. Note: this data is not used in the production track so not needed anyway